smart/plugin.go

```diff
@@ -166,6 +177,10 @@ func (sc *SmartCollector) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricTy
 
 //GetConfigPolicy returns a ConfigPolicy
 func (p *SmartCollector) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
-	c := cpolicy.New()
-	return c, nil
+	cp := cpolicy.New()
+	rule, _ := cpolicy.NewStringRule("proc_path", false, "/proc")
```

Comment:
@marcintao perfectly right. However the adaption of plugins  for complying with docker environment is usually done in 2 steps:

1. uniformization of plugin configuration

2. make usage of the configuration to get access to host from within container